discovery-provider/src/utils/default_underground_trending_tracks_strategy.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
```

Comment:
File structure wise, would be nice to colocate all of the strategies under a trending strategy folder somewhere